guides/common/attributes-titles-base.adoc

```diff
@@ -0,0 +1,25 @@
+// Base for titles
+:AdministeringDocTitle: Administering {ProjectName}
+:AdministeringAnsibleDocTitle: Managing {Project} with Ansible
+:AppCentricDeploymentDocTitle: Deploying Hosts using Application Centric Approach
+:ConfiguringAnsibleDocTitle: Configuring {Project} to use Ansible
+:ConfiguringLoadBalancerDocTitle: Configuring {SmartProxies} with a Load Balancer
+:ContentManagementDocTitle: Managing Content
+:ConvertinHostRHELDocTitle: Converting a Host to RHEL
+:DeployingAWSDocTitle: Deploying {ProjectName} on Amazon Web Services
+// InstallingServer - defined per product
+//:InstallingServerDocTitle: Installing {ProjectServerTitle} - base unused
+// Installing Disconnected - defined for Satellite only
+//:InstallingServerDisconnectedDocTitle: base unused
+:InstallingSmartProxyDocTitle: Installing an External {SmartProxyServerTitle} {ProjectVersion}
+// Puppet Guide - overridden in Satellite
+:ManagingConfigurationsPuppetDocTitle: Configuring Hosts Using Puppet
+:ManagingHostsDocTitle: Managing Hosts
+:ManagingOrganizationsLocationsDocTitle: Managing Organizations and Locations in {Project}
+:PlanningDocTitle: Planning for {ProjectName}
+:ProvisioningDocTitle: Provisioning Hosts
+// Quickstart - defined for Foreman-DEB, Foreman-EL, and Katello
+//:QuickstartDocTitle: Quickstart Guide - base unused
+// Release Notes - defined for Foreman and Katello
+//:ReleaseNotesDocTitle: Release Notes - base unused
```

Comment:
What are those comments for? If it's unused, I think we should remove it.

Reply:
I thought it could be useful to have a complete list of these variables in one place for knowledge-keeping purposes.

Reply:
It's kinda supposed to be documentation of the title attributes.

Reply:
Let's delete. We have older branches if we need to reference. This could cause confusion. The general policy is not to comment out but remove.

Reply:
Oh I see what you're doing now. Ignore above comment but perhaps we move this to the readme and then point to the readme ?

Reply:
OK, then keep the comments.

Reply:
I think it's better to have comments than moving the intel into a separate file. Keeping the comments.

Reply:
Totally fine with me